docs/internals/howto-release-django.txt

```diff
@@ -50,22 +50,50 @@ There are a lot of details, so please read on.
 Prerequisites
 =============
 
-You'll need a few things before getting started:
+You'll need a few things before getting started. If this is your first release,
+you'll need to coordinate with another releaser to get all these things lined
+up.
```

Comment:
```suggestion
you'll need to coordinate with another releaser, and write to the Ops mailing
list requesting the required access and permissions.
```
Up to you, I think it flows better to be in the same paragraph rather than have this in the admonition below, as I believe before "coordinate with another releaser" was supposed to cover this